dlt/sources/rest_api/__init__.py

```diff
@@ -405,14 +410,22 @@ def paginate_dependent_resource(
 
 def _validate_config(config: RESTAPIConfig) -> None:
     c = deepcopy(config)
-    client_config = c.get("client")
+    _mask_client_config_auth(c.get("client"))
+    resources = c.get("resources")
```

Comment:
if you do `c.get("resources", [])`, you can skip the `if resources:`

Reply:
changed in 1bf1fe766ab071f6a5df70f9434e0ae2b8c984ba